keyless-entry

```diff
@@ -37,7 +37,12 @@
 # accessible on boot and executable or it won't work.
 #
 # You will be prompted to enter an existing LUKS key (passphrase)
-# once for each encrypted device during configure.
+# once for each encrypted device during configure. This will add a 
```

Comment:
There is trailing whitespace on this line. Please make sure that your code passes flake8 before submitting your PR. (There are other flake8 violations in your version of the code which I am not going to point out one by one; just please make sure the script is flake8-clean before submitting it.)